packages/@aws-cdk/aws-apprunner-alpha/lib/service.ts

```diff
@@ -1259,6 +1267,10 @@ export class Service extends cdk.Resource implements iam.IGrantable {
       healthCheckConfiguration: this.props.healthCheck ?
         this.props.healthCheck.bind() :
         undefined,
+      observabilityConfiguration: props.observabilityConfiguration ? {
+        observabilityEnabled: true,
+        observabilityConfigurationArn: props.observabilityConfiguration?.observabilityConfigurationArn,
```

Comment:
```suggestion
        observabilityConfigurationArn: props.observabilityConfiguration.observabilityConfigurationArn,
```
We already checked for its existence above.